### PR TITLE
[artifactory] Upgrade postgres version to 10.13.0

### PR DESCRIPTION
**Pull request by amithins**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated

**What this PR does / why we need it**:

Upgrade Postgresql version of artifactory to 10.13.0.

### CODE REVIEW

**Comment by eldada** [comment]
Fix the CI

File: stable/artifactory-ha/CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,16 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+<<<<<<< HEAD
```
Comment (danielezer) [inline stable/artifactory-ha/CHANGELOG.md:4]: @amithins you have a conflict here. Please fix it

Comment (chukka) [reply stable/artifactory-ha/CHANGELOG.md]: Fixed

**Comment by eldada** [comment]
I suggest you add a section in https://github.com/jfrog/charts/blob/master/stable/artifactory/UPGRADE_NOTES.md for the details and process of upgrading with the bundled PG. It's a fair copy-paste of the process with some updates.

**Comment by chukka** [comment]
> I suggest you add a section in https://github.com/jfrog/charts/blob/master/stable/artifactory/UPGRADE_NOTES.md for the details and process of upgrading with the bundled PG. It's a fair copy-paste of the process with some updates.

@eldada Added upgrade notes - please review

**Pull request merged**